metadata/com.zaxxer/HikariCP/5.0.1/reflect-config.json

```diff
@@ -171,6 +171,10 @@
         "name": "getDriverClassName",
         "parameterTypes": []
       },
+      {
+        "name": "getExceptionOverrideClassName",
```

Comment:
Since there are no new tests provided in this PR, where do these entries come from?

Reply:
From the following change:
`<logger name="com.zaxxer.hikari.HikariConfig" level="DEBUG"/>`

The `logConfiguration` method from `HikariConfig` tries to load all properties using reflection and if there are no these additional metadata entries, the hikari connection pool fails in a native image.